tools/explorer/test/run_tests.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import model_explorer
+
+import requests
+import time
+import multiprocessing
+import pytest
+import glob
+
+HOST = "localhost"
+PORT = 8002
+CONVERT_URL = "http://" + HOST + ":" + str(PORT) + "/apipost/v1/send_command"
+TEST_LOAD_MODEL_PATHS = [
+    "test/ttmlir/Dialect/TTNN/mnist_sharding.mlir",
+    "tools/explorer/test/models/*.mlir",
+]
+
+
+def get_test_files():
+    files = []
+    for path in TEST_LOAD_MODEL_PATHS:
+        files.extend(glob.glob(path))
+    return files
+
+
```

Comment:
We should set up a workflow to run these tests on PR. I had something similar for infra python scripts

https://github.com/tenstorrent/tt-forge-fe/blob/main/.github/workflows/infra-test.yml
runs tests from infra subfolder in case there are changes on "infra" path
https://github.com/tenstorrent/tt-forge-fe/tree/main/infra